### PR TITLE
Docs: Installation guide

### PR DESCRIPTION
**Pull request by BullHorn7**
First draft.
Need to add PWS guide
Verify nothing important is missing

### CODE REVIEW

File: documentation/user/Installation.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+order: 0
+parent: wiki
+---
+
+## 1. Overview
+This guide will explain how to get ACE3 up and running faster than you can say banana.
+
+## 2. Prerequisites
+- Download [CBA](http://www.armaholic.com/page.php?id=18767).
+- Download [ACE3](https://github.com/acemod/ACE3/releases/tag/v3.0.1).
```
Comment (PabstMirror) [inline documentation/user/Installation.md]: https://github.com/acemod/ACE3/releases/latest should point to most recent release without needing to be updated each time.

Comment (BullHorn7) [inline documentation/user/Installation.md]: Done, thanks.

File: documentation/user/Installation.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+layout: wiki
+title: Installation
+description: Step-by-step guide for installing ACE3
+group: user
+order: 0
+parent: wiki
+---
+
+## 1. Overview
+This guide will explain how to get ACE3 up and running faster than you can say banana.
```
Comment (bux578) [inline documentation/user/Installation.md]: :+1:

**Comment by bux578** [comment]
Could you rename the file to `installation-guide.md` and change the title accordingly to `Installation Guide`.

Pictures would be awesome (that's a liitle bit tricky with this documentation sync github thingy, but would be awesome on the hompage. Ping me on slack and I show you how to do it)

**Comment by BullHorn7** [comment]
Sure, after work. I can take pictures easily, you'll just have to show me how to host them on github (otherwise it'll be on imgur or similar)

File: documentation/user/installation-guide.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+### 4.1 Steam Launch Parameters
+- Not supported with the new Arma 3 Launcher
+
+### 4.2 Arma3Sync
+- Inspect `Addon Options` and verify the `Addon Search Directories` are leading to your Arma 3 folder.
+- Return to the `Addons` tab and click `Refresh`.
+- Drag and drop `@ace` and `@CBA_A3` from `Available Addons` into the `Addon Groups` list and make sure the checkboxes are ticked.
+- Click `Start Game`
+
+### 4.3 Play withSix
```
Comment (Tourorist) [inline documentation/user/installation-guide.md]: Play withS**IX** - proper name stylization.

**Pull request merged**